README.md

```diff
@@ -1,3 +1,36 @@
 # cluster-baremetal-operator
-This repository is for a second level operator for baremetal installs. 
-See https://github.com/openshift/enhancements/blob/master/enhancements/baremetal/an-slo-for-baremetal.md for details about this operator.
+
+Introduction
+
+OpenShift supports multiple cloud platform types like AWS, GCP, Azure etc. In addition to these public cloud platforms, OpenShift can also be used to set up a fully functional OpenShift cluster with just Baremetal servers which are either pre-provisioned or are fully provisioned using OpenShift’s Baremetal IPI solution.
+The CBO fits into this specific solution by deploying on an OpenShit control plane all the necessary components required to take an unprovisioned server to a fully functional worker node ready to run OpenShift compute workloads. This second level operator was written based on enhancement :
+https://github.com/openshift/enhancements/blob/master/enhancements/baremetal/an-slo-for-baremetal.md for details about this operator.
+
+What does it do?
+
+Cluster-baremetal-operator (CBO) is designed to be an OpenShfit Operator that is responsible for deploying all components required to provision Baremetal servers into worker nodes that join an OpenShift cluster. 
```

Comment:
s/OpenShfit/OpenShift

Reply:
yeah and on line 6 check that spelling too :-O

Reply:
Thanks! Fixed!